### ISSUE TITLE
Select Component - label is orphaned from element in WAVE tool

### ISSUE THREAD

**Issue by sane6** [openvi-foundation/openvue#184]
### Describe the bug

In the latest version of PrimeVue, there is no proper way to attach a <label> element to the <Select> component. You'll see in the docs that the WAVE (web accessibility evaluation tool) that these labels are being orphaned, even when using the <label for="xyz" />XYZ</label><Select inputId="xyz" /> format as shown in the demo:

<img width="3978" height="2330" alt="Image" src="https://github.com/user-attachments/assets/276c556b-c542-450f-8daa-91a128a890d0" />

### Reason for not contributing a PR

- [x] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/primevue-4-vite-issue-template-5wryvgrx?file=src%2FApp.vue

### Environment

.

### Vue version

3.5.18

### PrimeVue version

4.3.6

### Node version

22.16.0

### Steps to reproduce the behavior

Create any select element as shown with a proper label. Observe that the two are not connected and report "orphaned form label" in accessibility testing tools.

### Expected behavior

I'd expect these not to throw "orphaned form label" errors in accessibility testing.

Copied to track in our fork; not affiliated with the original report being closed/resolved upstream

Originally reported at: [primefaces#7961](https://github.com/primefaces/primevue/issues/7961)
Originally reported by: https://github.com/adamshafer
